doc/_extensions/zephyr/application.py

```diff
@@ -186,6 +189,18 @@ def run(self):
         if compact and skip_config:
             raise self.error('Both compact and maybe-skip-config options were given.')
 
+        if zephyr_app:
+            # as folks might use "<...>" notation to indicate a variable portion of the path, we
+            # deliberately don't check for the validity of such paths.
+            if "<" not in zephyr_app and ">" not in zephyr_app:
```

Comment:
Allows to filter for even more characters.
```suggestion
            if not any(x in zephyr_app for x in ('<', '>')):
```